ambry-store/src/main/java/com/github/ambry/store/IndexSegment.java

```diff
@@ -105,8 +104,8 @@ class IndexSegment implements Iterable<IndexEntry> {
   private int indexSizeExcludingEntries;
   private int firstKeyRelativeOffset;
   private IFilter bloomFilter = null;
-  private int valueSize;
-  private int persistedEntrySize;
+  private int valueSize = VALUE_SIZE_INVALID_VALUE;
```

Comment:
what is valueSize ? Size of the value in the key-value pair in the index ?

Reply:
It's the size of the IndexValue, yes, it's the size of the value in the key-value pair.